app/models/users/scopes/find_by_login.rb

```diff
@@ -34,8 +34,17 @@ module FindByLogin
     extend ActiveSupport::Concern
 
     class_methods do
-      def self.find_by_login(login)
-        where(["LOWER(login) = ?", login.to_s.downcase]).first
+      def by_login(login)
```

Comment:
I think `with_login` might've been a better name. But this is not great enough a peeve to not merge this PR.